src/Uuid.php

```diff
@@ -24,6 +28,31 @@ public static function bootUuid()
 
     protected function generateUuid()
     {
-        return RamseyUuid::uuid4()->toString();
+        try {
+            switch ($this->uuid_version) {
+               case 1:
+                    return RamseyUuid::uuid1()->toString();
+                    break;
+
+               case 3:
+                    return RamseyUuid::uuid3(RamseyUuid::NAMESPACE_DNS, $this->uuid_string)->toString();
+                    break;
+
+               case 4:
+                    return RamseyUuid::uuid4()->toString();
+                    break;
+
+               case 5:
+                    return RamseyUuid::uuid5(RamseyUuid::NAMESPACE_DNS, $this->uuid_string)->toString();
+                    break;
+
+               default:
+                    break;
+           }
+        } catch (UnsatisfiedDependencyException $e) {
+            // Some dependency was not met. Either the method cannot be called on a
+            // 32-bit system, or it can, but it relies on Moontoast\Math to be present.
+            return 'Caught exception: '.$e->getMessage()."\n";
```

Comment:
I'd say we leave it be caught by the Laravel error handler. To me, having this kind of issue must completly block your app's use.